testfixtures/popen.py

```diff
@@ -69,6 +69,11 @@ def Popen(self, args, bufsize=0, executable=None,
 
         self.stdout, self.stderr, self.returncode, pid, poll = behaviour
         self.poll_count = poll
+
+        if stderr == STDOUT:
+            self.stdout += self.stderr
+            self.stderr = b''
```

Comment:
not sure this is correct:
```
>>> p = Popen('echo hello 1>&2', shell=True, stdout=PIPE, stderr=STDOUT)
>>> print p.stderr
None
```

Would be great if `f.write(getattr(self, name))` below could also change to:

```python
value = getattr(self, name)
if value is None:
    continue
f = TemporaryFile()
f.write(value)
```

...and of course matching assertions in the test along the lines of `assert process.stderr is None`.